src/index.html

```diff
@@ -5,10 +5,77 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>John Roberts' CV</title>
-    <link rel="stylesheet" href="./css/index.scss" />
+    <link rel="stylesheet" href="css/index.css" />
```

Comment:
Actually, “SCSS” (for “Sassy CSS”), and is a superset of CSS3's syntax. This means that every valid CSS3 stylesheet is valid SCSS as well. So, to implement your CSS statement, you can keep using the SCSS file.

Reply:
Thanks for your reply. Can we reference SCSS files directly in an HTML file? Or we should transform them into CSS first? Cheers.

Reply:
Yes, [ParcelJS](https://parceljs.org/) is responsible for automatically transformed the SCSS. Here is the magic of launching your local dev server: https://github.com/nzittraining/cv-assignment/blob/master/package.json#L8